backend/app/adapter/graphql/resolver/changelog.go

```diff
@@ -0,0 +1,30 @@
+package resolver
+
+import (
+	"time"
+
+	"github.com/short-d/short/app/adapter/graphql/scalar"
+	"github.com/short-d/short/app/entity"
+)
+
+type ChangeLog struct {
+	changeLog []*Change
+}
+
+func (c ChangeLog) Changes() []*Change {
+	return c.changeLog
+}
+
+func (c ChangeLog) LastViewedAt() *scalar.Time {
+	currentTime := time.Now()
```

Comment:
There should be a `lastViewedAt` field in `ChangeLog` struct. That's the time the user previous viewed the change log.